src/components/network-area-diagram-viewer/network-area-diagram-viewer.ts

```diff
@@ -189,16 +208,16 @@ export class NetworkAreaDiagramViewer {
         // add events
         if (enableNodeMoving) {
             this.svgDraw.on('mousedown', (e: Event) => {
-                this.startDrag(e);
+                this.handleStartDrag(e);
             });
             this.svgDraw.on('mousemove', (e: Event) => {
-                this.drag(e);
+                this.handleDrag(e);
             });
             this.svgDraw.on('mouseup', (e: Event) => {
-                this.endDrag(e);
+                this.handleEndDrag(e);
             });
             this.svgDraw.on('mouseleave', (e: Event) => {
-                this.endDrag(e);
```

Comment:
here we know that e a MouseEvent, maybe it is better to do the cast here , so you don't need to do it later in all the usage of this event `handleStartDrag` `processStartDrag`...

`private handleStartDrag(event: MouseEvent) `
this remark is not mondatory to fix right now

Reply:
I will not touch this part, this is not my code. I only renamed the functions to make them easier to understand.

Reply:
I tried it and it is confusing. Moreover, this is not my code, I only renamed the functions to make them easier to understand.
Therefor, I will not change this in this PR.